Pipfile

```diff
@@ -57,6 +57,7 @@ pytest-cov = "==2.9.0"
 pytest-custom-exit-code = "==0.3.0"
 pytest-randomly = "==3.4.0"
 pytest-rerunfailures = "==9.0"
+GitPython = "==3.1.27"
```

Comment:
where do I need to add this to not receive this error?

```
Traceback:
tests/test_packages/test_connections/test_p2p_libp2p/test_go_code_matching_acn.py:30: in <module>
    import git
E   ModuleNotFoundError: No module named 'git'
```

Reply:
Most probably tox config since we run our tests using tox env.